apps/studio/src/App.css

```
@@ -1,2 +1,3 @@
 @tailwind base;
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)